app/presenters/auction_status_presenter_factory.rb

```diff
@@ -55,10 +72,86 @@ def end_date
   end
 
   def sign_in_link
-    link_to 'Sign in', sign_in_path
+    Url.new(link_text: 'Sign in', path_name: 'sign_in')
   end
 
   def sign_up_link
-    link_to 'sign up', sign_up_path
+    Url.new(link_text: 'sign up', path_name: 'sign_up')
+  end
+end
+
+class FutureUserIsGuest
+  attr_reader :auction
+
+  def initialize(auction:)
+    @auction = auction
+  end
+
+  def to_s
```

Comment:
one thing I do like about I18n is that it makes changing copy easier in the land of testing. Rather than changing a word and then breaking a bunch of high-level tests (which has happened to me this week), you change the translation and the key stays the same so no test changes are needed.